cell_classification/segmentation_data_prep.py

```diff
@@ -100,7 +97,12 @@ def get_instance_masks(self, data_folder, cell_mask_key):
             np.array:
                 The instance mask
         """
-        return np.zeros([500, 500]), np.zeros([500, 500])
+        instance_mask = imread(
+            os.path.join(data_folder, self.segmentation_fname + ".tiff")
```

Comment:
Your default formatter probably has 60 characters as the limit, but we use 99 characters in the repo as our limit. See if you can adjust it so that when it autoformats it doesn't try and fit everything in only 60

Reply:
It had 88 and I changed it to 99. Thanks for the suggestion. It now only adds excessive line breaks for arguments in function definitions.